xarray/plot/plot.py

```diff
@@ -239,10 +239,6 @@ def line(darray, *args, row=None, col=None, figsize=None, aspect=None,
                          'Passed DataArray has {ndims} '
                          'dimensions'.format(ndims=ndims))
 
-    # Ensures consistency with .plot method
-    if args is ():
-        args = kwargs.pop('args', ())
```

Comment:
I'm not quite sure why, but this seems to be important. Now five matplotlib tests are failing.

Reply:
Ah. Because you can have
```
>>> def f(*args, **kwargs):
...    return args, kwargs
    
>>> f(1, 2, args=(3, 4))
(1, 2) {'args': (3, 4)}
```

Reply:
I figured it out, it's because line() calls itself
line() -> _easy_facetgrid() -> map_dataarray_line() -> line()
I tried changing it to something clearer but a lot more stuff broke. I suggest leaving it there for now and taking care of it separately. For now I just clarified.